<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1552</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="metiers">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;file:///C:/Users/pc/Downloads/pexels-steve-johnson-1044989.jpg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(151, 175, 227);



</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <attribute name="title">
        <string>CRUDs</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>10</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Gestion des invités </string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>490</y>
          <width>451</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier</string>
        </property>
        <widget class="QPushButton" name="pb_modifier">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>191</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>130</y>
           <width>451</width>
           <height>131</height>
          </rect>
         </property>
         <property name="title">
          <string>Modifier</string>
         </property>
         <widget class="QPushButton" name="pb_modifier_3">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>50</y>
            <width>191</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>451</width>
          <height>421</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter un invité</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>       CIN</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>170</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>    Prénom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string> Date de naissance</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>290</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>    Adresse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>    Num tel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>370</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>       Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>    Etat civil</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ladresse">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>290</y>
           <width>271</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QComboBox" name="letat_civil">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>250</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_date_de_naissance">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>210</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>170</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>130</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);

</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_Num_tel">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>90</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_CIN">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>50</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="date_event">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>330</y>
           <width>271</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>330</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Date_event</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_historique">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>370</y>
           <width>112</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>Mise a jour</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>600</y>
          <width>451</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer selon CIN</string>
        </property>
        <widget class="QLineEdit" name="le_id_supp">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_supprimer">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>60</y>
          <width>1031</width>
          <height>801</height>
         </rect>
        </property>
        <property name="title">
         <string>Afficher les invités</string>
        </property>
        <widget class="QTableView" name="tab_invite">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>1011</width>
           <height>751</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>710</y>
          <width>451</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>Recherche et trie</string>
        </property>
        <widget class="QPushButton" name="pb_recherche">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Recherche</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="rech">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_tri_cin">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Tri selon CIN</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_tri">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>70</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_tri_adresse">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Trie selon Adresse</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_tri_2">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>110</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>métiers</string>
       </attribute>
       <widget class="QFrame" name="horizentalframe">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>100</y>
          <width>641</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5"/>
       </widget>
       <widget class="QFrame" name="Donutchart">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>100</y>
          <width>591</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>660</y>
          <width>1491</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>Remarques à noter:</string>
        </property>
        <widget class="QTextEdit" name="textEdit_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>1471</width>
           <height>141</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>291</width>
          <height>431</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>QR code</string>
        </property>
        <widget class="QLineEdit" name="qr">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_qr_code">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>50</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>QR code</string>
         </property>
        </widget>
        <widget class="QLabel" name="qrlabel">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>130</y>
           <width>191</width>
           <height>241</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="pb_pdf_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>510</y>
          <width>291</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Générer en PDF</string>
        </property>
        <widget class="QCommandLinkButton" name="pb_pdf">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>241</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Impression du contrat</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pb_arret">
        <property name="geometry">
         <rect>
          <x>1360</x>
          <y>860</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 186, 61);
</string>
        </property>
        <property name="text">
         <string>Arrêter</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
